README.md

```diff
@@ -24,11 +24,15 @@ You'll need to use some sort of strategy to export the environment variables in
 git clone git@github.com:18F/tock-management-reports.git
 cd tock-management-reports
 cp .env.example .env
-pyvenv env
+pyenv env
```

Comment:
This is actually [pyvenv](https://docs.python.org/3/library/venv.html) which comes native as of python 3.3 and is apparently deprecated as of 3.6. Now it should read `python -m venv env`.